### PR TITLE
MFTF: Admin Deletes Customer's Wishlist Item Test

### LINKED ISSUE #28721: [Issue] MFTF: Admin Deletes Customer's Wishlist Item Test
**Preconditions:**

This issue is automatically created based on existing pull request: magento/magento2#28632: MFTF: Admin Deletes Customer's Wishlist Item Test

---------
This PR contains a test for deleting items from customer wishlist as Admin user.

**Steps to reproduce:**

1 - Add a created product into wishlist as a customer
2 - Login as admin user
3 - Find created customer on grid
4 - Find added wishlist item
5 - Delete wishlist item
6 - Perform assertions
7 - Go to Storefront -> My Wishlist

**Actual Result:**

**Expected Result:**
The product deleted through the admin panel should not be in Wishlist

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
This PR contains a test for deleting items from customer wishlist as Admin user.

**Steps to reproduce:**

1 - Add a created product into wishlist as a customer
2 - Login as admin user
3 - Find created customer on grid
4 - Find added wishlist item
5 - Delete wishlist item
6 - Perform assertions

### Resolved issues:
1. [x] resolves magento/magento2#28721: MFTF: Admin Deletes Customer's Wishlist Item Test

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento run all tests

File: app/code/Magento/Customer/Test/Mftf/Section/AdminCustomerWishlistSection.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdminCustomerWishlistSection">
+        <element name="productName" type="input" selector="#wishlistGrid_filter_product_name"/>
+        <element name="searchButton" type="button" selector=".action-default.scalable.action-secondary"/>
+        <element name="deleteButton" type="text" selector=".even > td:nth-child(7) > a:nth-child(1)"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Customer/Test/Mftf/Section/AdminCustomerWishlistSection.xml]: This is very flaky way to find the element. 
If there is no other option, even selecting by label is more appropriate

File: app/code/Magento/Wishlist/Test/Mftf/Test/AdminDeleteCustomerWishListItemTest.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <stories value="Wishlist items deleting"/>
+            <title value="Admin deletes an item from customer wishlist"/>
+            <description value="Admin Should be able delete items from customer wishlist"/>
+            <group value="wishlist"/>
+        </annotations>
+        <before>
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
```
Comment (lbajsarowicz) [inline app/code/Magento/Wishlist/Test/Mftf/Test/AdminDeleteCustomerWishListItemTest.xml:24]: Please bear in mind that tests might be run in "on schedule" indexing mode

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
I see two elements that make this test flaky. Please take a look and adjust these 👍

File: app/code/Magento/Customer/Test/Mftf/Section/AdminCustomerWishlistSection.xml (hunk 1 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Customer/Test/Mftf/Section/AdminCustomerWishlistSection.xml]: Refactored

File: app/code/Magento/Wishlist/Test/Mftf/Test/AdminDeleteCustomerWishListItemTest.xml (hunk 2 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Wishlist/Test/Mftf/Test/AdminDeleteCustomerWishListItemTest.xml:24]: Added cron cli

**Comment by lbajsarowicz** [comment]
@magento run all tests

I'm going to review the PR in an hour or two

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Well done! Thank you for contribution.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7674](https://jira.corp.magento.com/browse/ENGCOM-7674) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
@magento create issue

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by slavvka** [comment]
@magento run all tests

**Pull request merged**